deploy.sh

```diff
@@ -79,6 +82,14 @@ if [ -n "${HELM_REPOSITORY}" ]; then
     fi
 fi
 
+# If OCI registry, we need to login before performing any action
+
+if [ "${OCI_REGISTRY}" == "true" ]; then
+   echo "${REPO_PASSWORD}" | helm registry login "${HELM_REPOSITORY}" --username "${REPO_USERNAME}" --password-stdin
```

Comment:
`helm registry login` expects different parameters than `helm repo add`
The former expects a host (i.e. ghcr.io) and the latter the URL of a repo where you can find an `index.yaml`